### PR TITLE
Upgrade 'standard' and run 'standard --fix'

### PR DESCRIPTION
**Pull request by abetomo**

### CODE REVIEW

File: lib/copy-sync/__tests__/copy-sync-prevent-copying-into-itself.test.js [hunk 1]
```diff
@@ -60,59 +60,59 @@ describe('+ copySync() - prevent copying into itself', () => {
...
         const dest = path.join(TEST_DIR, src.replace(/^\w:/, ''))
         return testSuccess(src, dest)
       })
-      it(`of itself`, () => {
+      it('of itself', () => {
         const dest = path.join(src, 'dest')
         return testError(src, dest)
       })
-      it(`should copy the directory successfully when dest is 'src_dest'`, () => {
+      it('should copy the directory successfully when dest is \'src_dest\'', () => {
```
Comment (RyanZim) [inline lib/copy-sync/__tests__/copy-sync-prevent-copying-into-itself.test.js]: Does the new standard version allow this?

```suggestion
      it("should copy the directory successfully when dest is 'src_dest'", () => {
```

Comment (abetomo) [reply lib/copy-sync/__tests__/copy-sync-prevent-copying-into-itself.test.js]: Thank you for the review.
No error with double quotes.
it changed.

Comment (RyanZim) [reply lib/copy-sync/__tests__/copy-sync-prevent-copying-into-itself.test.js]: OK, I realize this is a ton of work, but there's quite a few instances of this. I very much dislike backslash-escaped quotes `\'`, if we could use double-quotes there instead? If you don't want to do it, I'll get to it sooner or later.

Comment (abetomo) [reply lib/copy-sync/__tests__/copy-sync-prevent-copying-into-itself.test.js]: Changed to double quotes.

**Pull request merged**